### PR TITLE
Scripts for production deployments

### LINKED ISSUE #4382: Improve docker-based environment/deployment
- [x] #4276 Default to Python 3.8.6
- [x] #4391 Remove `/openlibrary` volume mount from docker-compose.production.yml
 - [x] #4375 Add vim and emacs to Dockerfile.olbase
 - [ ] #4395 Scripts for production deployments

[Deployment Scratchpad](https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad)

### PR DESCRIPTION
**Pull request by cclauss**
Closes #4382
This is a temporary script-based Open Library production deployment that will be refactored into `fabfile.py`.
It allows us to make sure all steps are clear and complete before migrating them into `fabfile.py`.
https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad
* __ol-home0:__ `time start_production_deploy.sh`
    1. __ol-home0, ol-covers0, web{1, 2}:__ `continue_production_deploy.sh`
    2. _User presses return when ready..._
    3. __ol-home0, ol-covers0, web1:__ `finish_production_deploy.sh`
    4. _User presses return when ready..._
    5. __ol-web2:__ `finish_production_deploy.sh`

### Testing

https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad

### CODE REVIEW

**Comment by cclauss** [comment]
Roadmap... (This PR or future iterations)
* [ ] rsync the four repos from ol-home0
* [ ] booklending_utils is only needed on ol-web{1, 2}
* [ ] ssh the commands
* [ ] uncomment cronjobs importbot solr-updater

**Review by mekarpeles — APPROVED** [review]
lgtm; 
- questions about how volume-mount affects ability to rollback
- questions about having web restarts occur in deployment script itself (currently we deploy and then restart independently with the intent that a human operator is available to rollback after the first failure experienced during blue/green)

File: scripts/deployment/finish_production_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+cd /opt/openlibrary
+export HOSTNAME="${HOSTNAME:-$HOST}"
+export COMPOSE_FILE="docker-compose.yml:docker-compose.production.yml"
+# WARNING! Moment of downtime 😬 
+docker-compose down
+docker volume rm openlibrary_ol-vendor openlibrary_ol-build openlibrary_ol-nodemodules
+if [[ $HOSTNAME == ol-covers0.* ]]; then
+    docker-compose up -d --scale covers=2 covers_nginx memcached
+elif [[ $HOSTNAME == ol-home0.* ]]; then
+    docker-compose up -d --no-deps infobase infobase_nginx affiliate-server  # cronjobs importbot solr-updater
```
Comment (cdrini) [inline scripts/deployment/finish_production_deploy.sh:18]:
```suggestion
    docker-compose up -d --no-deps infobase infobase_nginx affiliate-server solr-updater # cronjobs importbot
```

**Comment by mekarpeles** [comment]
@cclauss and I agreed that given this is the process we're currently using, we can go ahead and land this initial set of changes and open an issues for behavior change which needs to occur as a followup.

**Pull request merged**